website/addons/base/testing/logger.py

```diff
@@ -42,19 +30,3 @@ def test_log_deauthorized_when_node_settings_are_deleted(self):
 
         last_log = project.logs[-1]
```

Comment:
This test `test_log_deauthorized_when_node_settings_are_deleted` still uses `utils.BoxNodeLogger` which has been removed.